api/src/services/listing.service.ts

```diff
@@ -654,10 +682,29 @@ export class ListingService implements OnModuleInit {
         jurisdictionId: dto.jurisdictions.id,
       },
     );
+    if (dto.units && dto.unitGroups) {
+      throw new BadRequestException('Cannot provide both units and unitGroups');
+    }
+
+    const rawJurisdiction = await this.prisma.jurisdictions.findFirst({
+      where: {
+        id: dto.jurisdictions.id,
+      },
+      include: {
+        featureFlags: true,
+      },
+    });
+
+    const enableUnitGroups = rawJurisdiction?.featureFlags?.find(
+      (featureFlag) => featureFlag.name === 'enableUnitGroups',
```

Comment:
Chore: My PR was merged this morning after you created this PR so this is mostly just a note. I created a new enum called `FeatureFlagEnum` where we can add the name of each feature flag. Once you rebase you could add this flag name there and replace all places of `enableUnitGroups` with the enum.